pkg/web_app/lib/src/account.dart

```diff
@@ -516,29 +497,62 @@ class _ConsentWidget {
     if (!window.confirm('Are you sure you want to accept?')) {
       return;
     }
-    try {
+    await _rpc(() async {
       final rs = await client.resolveConsent(
           pageData.consentId, ConsentResult(granted: true));
       _updateButtons(rs.granted);
-    } on RequestException catch (e) {
-      final map = e.bodyAsJson();
-      window.alert(map['message'] as String);
-    }
+    });
   }
 
   Future _reject() async {
     if (!window.confirm('Are you sure you want to reject?')) {
       return;
     }
-    try {
+    await _rpc(() async {
       final rs = await client.resolveConsent(
           pageData.consentId, ConsentResult(granted: false));
       _updateButtons(rs.granted);
-    } on RequestException catch (e) {
-      final map = e.bodyAsJson();
-      window.alert(map['message'] as String);
-    }
+    });
   }
 
   bool get isActive => _content != null && _buttons != null;
 }
+
+Future<R> _rpc<R>(Future<R> fn()) async {
+  // Disable only the selected Elements that are not disabled at the moment.
```

Comment:
This doesn't only disable the _selected elements_ :)

This disables all input input elements that are not already disabled..

So you can't edit the description while the "send invite" operation is running...
I think this is fine... I'm not 100% sure.. but let's take it for a spin.

Maybe we want any operation that submits data, like "send invite" or "update description" to remove the entire tab... and run with a spinner being displayed... When completed then either an error is showed on the current page, or the user is redirect to a different page where the change is reflected... Or in the case of "send invite" a text says "invite sent" and a button says "continue", which then reloads the current page.

Reply:
I'm planning to do a modal dialog for error messages that would be in the front of the page. We could use the same thing for communicating that "something is ongoing, please wait".